### DISCUSSION TITLE
Volume Slider in Windows 11 Style

### DISCUSSION THREAD

**Discussion by 33979201** [valinet/ExplorerPatcher#1696] [Ideas]
Hi, thank you for creating this patcher, it has been working really well. However, there is one small problem. I used to have windows 11 volume slider at the bottom of the screen when I try to turn the volume up or down even when I have windows 10 style taskbar, so I could see the volume. After the new update, for some reason, the volume slider disappeared. I tried to uninstall and re-install but of no avail. I don't see a volume slider in windows 10 style, either. Is it possible to add the volume slider as a feature to turn on and off?

**Comment by totok9** [comment]
Hello,

Big up for your message.
I'have the same issue for about one month. Re-install audio driver and ExplorerPatcher are not helping.

**Comment by pyrates999** [comment]
Please post windows version and build.
Please post explorer patcher version.

**Reply by GLlZZYGUZZLER** [reply]
I have this issue too - no visible volume bar :[

- Windows 11 22H2 (OS build 22621.2134)

- ExplorerPatcher 22621.1992.56.1

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1692

Please close this one.
